### PR TITLE
Fix minikube 0.26.0 integration

### PR DESCRIPTION
**Pull request by aanm**
**Summary of changes**:

- Add etcd standalone spec file in the GSGs
- Modified k8s generator to have config maps always on top
- Add istio GSG to the list of GSGs

Fixes: #3672

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

File: examples/kubernetes/Makefile [hunk 1]
```diff
@@ -38,8 +38,9 @@ $(foreach K8S_VERSION,$(K8S_VERSIONS), \
 # Add all YAML files as dependencies of concatenated files.
 $(foreach K8S_VERSION,$(K8S_VERSIONS), \
 	$(foreach CONCAT_FILE,$(CONCAT_FILES), \
-		$(eval $(K8S_VERSION)/$(CONCAT_FILE): $(patsubst %,$(K8S_VERSION)/%,$(NON_DAEMONSET_YAMLS))) \
-		$(eval $(K8S_VERSION)/$(CONCAT_FILE): $(K8S_VERSION)/$(patsubst %.yaml,%-ds.yaml,$(CONCAT_FILE)))))
+		$(eval $(K8S_VERSION)/$(CONCAT_FILE): $(sort    $(patsubst %,$(K8S_VERSION)/%,$(NON_DAEMONSET_YAMLS)) \
```
Comment (rlenglet) [inline examples/kubernetes/Makefile:41]: 👍

**Review by rlenglet — APPROVED** [review]

**Comment by rlenglet** [comment]
How about we prefix the template files with numbers, e.g. `01-cilium-cm.yaml`, `99-cilium-ds.yaml`, to make the ordering more explicit?

File: examples/kubernetes/Makefile [hunk 2]
```diff
@@ -38,8 +38,9 @@ $(foreach K8S_VERSION,$(K8S_VERSIONS), \
...
+														$(K8S_VERSION)/$(patsubst %.yaml,%-ds.yaml,$(CONCAT_FILE)))) \
+	))
```
Comment (raybejjani) [inline examples/kubernetes/Makefile:43]: The indentation is a little messed up on this second line. They both use tabs so I'm not sure if it's a github UI artefact

**Review by raybejjani — APPROVED** [review]

**Pull request merged**